src/components/AmountFilter.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Button, Popover, InputLabel, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import RemoveIcon from '@material-ui/icons/Remove';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    height: 57,
+  },
+  labelStyle: { paddingLeft: theme.spacing(1) },
+  dashStyle: { margin: '45px auto auto' },
+  filterStyle: { margin: `${theme.spacing(1)}px 0` },
+}));
+
+export default function AmountFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [values, setValues] = useState({});
+
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const handleChange = (index, val) =>
+    setValues({ ...values, [index]: parseInt(val, 10) || '' });
+
+  function displayFilter(label, value, onChange) {
+    return (
+      <Grid item className={classes.filterStyle}>
+        <InputLabel className={classes.labelStyle}>{label}</InputLabel>
+        <AmountTextField
+          disabled={false}
```

Comment:
Maybe `disabled={false}` should be the default? If it isn't I'd make it the default and change where it _is_ disabled to have the property `disabled`/`disabled={true}`.